pyproject.toml

```diff
@@ -37,7 +37,7 @@ dynamic = ["version", "dependencies", "readme"]
 # Specify the dependencies for the library from what is given in requirements.txt
 dependencies = {file = ["requirements.txt"]}
 
-readme = {file = ["README.md", "README_clients.md", "README_tests.md"]}
+readme = {file = ["README.md"]}
```

Comment:
Inclusion of multiple readme files was not allowing "python -m twine check <wheel_file>" to pass.
Go back to basics.